charts/chronograf/README.md

```diff
@@ -84,35 +76,39 @@ The following table lists the configurable parameters of the chronograf chart an
 | `tolerations`                | Toleration labels for pod assignment                                                                      | []                                          |
 | `affinity`                   | Affinity settings for pod assignment                                                                      | {}                                          |
 
-Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`. For example,
+To configure the chart, do either of the following:
 
-```bash
-helm upgrade --install my-release \
-  --set ingress.enabled=true,ingress.hostname=chronograf.foobar.com \
-    influxdata/chronograf
-```
+- Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`. For example, use the following command:
```

Comment:
to `helm upgrade --install`?

Reply:
Yes, lets always use `upgrade --install`

Reply:
updated